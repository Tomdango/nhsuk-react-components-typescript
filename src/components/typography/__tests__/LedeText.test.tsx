import React from 'react';
import LedeText from '../LedeText';
import { shallow } from 'enzyme';

describe('LedeText', () => {
  it('matches snapshot', () => {
    const element = shallow(<LedeText>Text</LedeText>);
    expect(element.text()).toBe('Text');
    expect(element).toMatchSnapshot();
    expect(element.hasClass('nhsuk-lede-text')).toBeTruthy();
    element.unmount();
  });
});
