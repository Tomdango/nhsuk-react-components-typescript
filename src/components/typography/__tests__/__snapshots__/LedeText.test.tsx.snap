// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LedeText matches snapshot 1`] = `
<p
  className="nhsuk-lede-text"
>
  Text
</p>
`;
